src/features/klant/KlantAanmaken.vue

```diff
@@ -96,6 +96,7 @@ import { createKlant } from "../contactverzoek";
 import SimpleSpinner from "../../components/SimpleSpinner.vue";
 import { useContactmomentStore } from "@/stores/contactmoment";
 import ApplicationMessage from "../../components/ApplicationMessage.vue";
+import router from "@/router";
```

Comment:
Interesting to see that this works too. However, I think it's better to stick to the convention:

```
import { useRouter } from 'vue-router'
const router = useRouter();
```

Reply:
Was an automatic import from my IDE, seeing your solution, I'm also surprised to see that that worked... 😄 

Fixed.